# Add EasyBuild paths for dependency searches
# August 22nd 2018 by B. Hajgato (Free University Brussels - VUB)
--- arpackpp-e7faceeafec9b88c2a9a79b2970506824a391735/CMakeLists.txt.orig	2018-03-25 03:56:33.000000000 +0200
+++ arpackpp-e7faceeafec9b88c2a9a79b2970506824a391735/CMakeLists.txt	2018-08-22 15:07:08.642684849 +0200
@@ -61,7 +61,7 @@
 # Find LAPACK and BLAS
 # OPENBLAS or ( ( SystemOpenblas or BLAS) and LAPACK)
 ## prefer local openblas
-find_library(OPENBLAS_LIB openblas PATHS external NO_DEFAULT_PATH)
+find_library(OPENBLAS_LIB openblas PATHS $ENV{EBROOTOPENBLAS}/lib $ENV{EBROOTOPENBLAS}/lib64 external NO_DEFAULT_PATH)
 IF (OPENBLAS_LIB)
   make_global_path(OPENBLAS_LIB "OpenBLAS Library")
   set(LAPACK_LIBRARIES ${OPENBLAS_LIB}) #local openblas has lapack build in
@@ -169,7 +169,7 @@
 # Suitesparse 
 if (UMFPACK OR CHOLMOD)
   # Suitesparse CHOLMOD
-  find_path(SUITESPARSE_DIR SuiteSparse_config/SuiteSparse_config.h "external/SuiteSparse")
+  find_path(SUITESPARSE_DIR SuiteSparse_config/SuiteSparse_config.h $ENV{EBROOTSUITESPARSE} "external/SuiteSparse" )
   IF (NOT SUITESPARSE_DIR)
     MESSAGE(STATUS "SuiteSparse Directory is required but could not be found")
     SET(ABORT_CONFIG TRUE)
@@ -180,7 +180,7 @@
   find_library(CHOLMOD_LIB libcholmod.a ${SUITESPARSE_DIR}/CHOLMOD/Lib  )
   find_library(COLAMD_LIB libcolamd.a ${SUITESPARSE_DIR}/COLAMD/Lib  )
   find_library(CCOLAMD_LIB libccolamd.a ${SUITESPARSE_DIR}/CCOLAMD/Lib  )
-  find_library(METIS_LIB libmetis.a ${SUITESPARSE_DIR}/metis-4.0 )
+  find_library(METIS_LIB libmetis.a ${SUITESPARSE_DIR} )
   find_library(CAMD_LIB libcamd.a ${SUITESPARSE_DIR}/CAMD/Lib )
   find_library(AMD_LIB libamd.a ${SUITESPARSE_DIR}/AMD/Lib )
 
